### PR TITLE
Images intro

### PR DESCRIPTION
**Pull request by carriewright11**
Adding the new images. @ehumph does this lead into the more complicated image OK? or do you think we need a bit more.

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -53,8 +53,15 @@ The course will cover:
 - Guidance on how to update a DMS plan after submission or funding
 - General tips and frequently asked questions
 
-```{r, echo=FALSE, fig.alt='Pathway of chapters covered in this course.', out.width = '100%'}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1luFoDzF6aDJEebbL6iWoJ_s8s9nQnaWLL5jghbmWdak/edit#slide=id.g16b04bd80e5_1_123")
+
+```{r, echo=FALSE, fig.alt='Main topics covered in this course: Introducing the DMS policy, Creating a DMS plan, DMS submission process.', out.width = '100%'}
```
Comment (avahoffman) [inline index.Rmd:57]: I feel like this image belongs right before "Motivation" H2. Thoughts?

**Review by avahoffman — APPROVED** [review]

**Comment by ehumph** [comment]
The new images look awesome!

File: index.Rmd (hunk 1 above)
Comment (ehumph) [reply index.Rmd:57]: I think @avahoffman is right.

Comment (carriewright11) [reply index.Rmd:57]: so you mean as the first thing for the next section?

**Comment by avahoffman** [comment]
> The new images look awesome!

agreed!

**Comment by carriewright11** [comment]
works for me!

**Comment by avahoffman** [comment]
Great! 🙌🏻

**Pull request merged**